### ISSUE TITLE
Wrong method implementation of count in sql.clj

### ISSUE THREAD

**Issue by saikatkumardey** [gorillalabs/sparkling#54]
currently implemented as:
```clojure

(defn count
  "grouped data count"
  [cols grouped-data]
  (.max grouped-data (into-array cols)))
```

It should be 

```clojure
(defn count
  "grouped data count"
  [cols grouped-data]
  (.count grouped-data (into-array cols)))
```

**Comment by chrisbetz** [comment]
Thanks,

I'll check and repair.

Bye

Chris

**Comment by chrisbetz** [comment]
done, merged, fixed in 2.1.2
